riscv/mmu.h

```diff
@@ -136,6 +137,11 @@ class mmu_t
     return load<T>(addr, {forced_virt, hlvx, lr});
   }
 
+  template<typename T>
+  T ss_load(reg_t addr) {
+    return load<T>(addr, {false, false, false, /*shstk=*/ true});
```

Comment:
ss_load and ss_store should also do a alignment check i.e. ssp aligned to sizeof(T) and cause a  store_access_fault if not aligned.